mcs/class/System.Net.Http/System.Net.Http/HttpClient.wasm.cs

```diff
@@ -9,41 +9,14 @@ public partial class HttpClient
 
 #pragma warning disable 649
         private static Func<HttpMessageHandler> GetHttpMessageHandler;
-        // Used to skip the type lookup on later calls.
```

Comment:
If you take this approach of making internals visible and then calling directly into `WasmHttpMessageHandler.GetHttpMessageHandler`, then you also do not need this `Func<HttpMessageHandler> GetHttpMessageHandler` anymore.

Reply:
This will be removed at a later date.  There are still users using this until we get them moved over.

Reply:
> The plan is still to remove the static variable but I think it makes sense to allow an an override so that we can allow experimentation outside the browser.